iperf-post-process

```diff
@@ -363,26 +366,37 @@ sub process_hunting_results {
         $ts_end=$times{'end'};
 
         while (<$fh>) {
+            #
+            # Note for hunter we do not process time series data, but process the final line of 
+            # the highest/best run. The "PASS: ... receiver" line.
+            #
```

Comment:
Does this mean that you are not indexing the individual samples?

Are you creating multiple periods (and a primary period for the actual measurement that is used) when doing the binary search?

Reply:
@atheurer ^^

Reply:
Yes we do not log_sample() for individual time-series data of each iperf run. You can think of one hunter run is one sample. Reason being one iperf run inside a hunter run is of DIFFERENT bitrate.  We will not be able to make sense if log them and query them late with --resolution as they will show numbers up or down randomly unless we tag them with bit rate. Other than that, we cannot aggregate these "samples" nor compute anything out from them.

Reply:
At a minimum, the final trial (the one which determined the result) should have all samples logged, and this trial should be the "primary period" for the sample.  Ideally, all previous trials should have their time-series samples logged as a non-primary-period (you can name them whatever you want, like search-1, search-2, etc).

Reply:
I wrote an issue for it https://github.com/perftool-incubator/bench-iperf/issues/23